dlls/player.h

```diff
@@ -322,6 +322,8 @@ class CBasePlayer : public CBaseMonster
 	char m_SbarString1[ SBAR_STRING_SIZE ];
 	
 	float m_flNextChatTime;
+
+	bool m_bSentBhopcap; // If false, the player just joined and needs a bhopcap message.
```

Comment:
It has been brought to my attention that I forgot to initialize this variable to `false` anywhere. Unfortunately I don't recall the order of function calls during an entity lifetime, but it should probably be set to `false` somewhere around `Spawn()` or `Precache()`. Maybe it could even be done right here (`bool m_bSentBhopcap = false;`), although I don't see any other variables being initialized like this, perhaps for a reason. It's also possible that whatever creates `CBasePlayer` entities zeroes out all of its members making this unnecessary.